src/main/java/com/fnp/checkexception/unchecked-exceptions-program/UncheckedException.java

```diff
@@ -0,0 +1,17 @@
+import customexception.*;
+
+class UncheckedException{	
```

Comment:
change this class name also @saranshvijay this seems to be like more of a exception name

Reply:
change the name of the folder also which contains this class.